modules/rosa-getting-started-install-configure-cli-tools.adoc

```diff
@@ -0,0 +1,159 @@
+====
+
+.Procedure
+
+. Install and configure the latest AWS CLI (`aws`).
+.. Follow the link:https://aws.amazon.com/cli/[AWS Command Line Interface] documentation to install and configure the AWS CLI for your operating system.
++
+Specify your `aws_access_key_id`, `aws_secret_access_key`, and `region` in the `.aws/credentials` file. See link:https://docs.aws.amazon.com/cli/latest/userguide/cli-configure-quickstart.html[AWS Configuration basics] in the AWS documentation.
++
+[NOTE]
+====
+You can alternatively use the `AWS_DEFAULT_REGION` environment variable to set the default AWS region.
+====
+.. Query the AWS API to verify if the AWS CLI is installed and configured correctly:
++
+[source,terminal]
+----
+$ aws sts get-caller-identity
+----
++
+.Example output
+[source,terminal]
+----
+<aws_account_id>    arn:aws:iam::<aws_account_id>:user/<username>  <aws_user_id>
+----
+
+. Install and configure the latest ROSA CLI (`rosa`).
+.. Navigate to the link:https://console.redhat.com/openshift/downloads[*Downloads*] page on the {OCM} and download the latest version of the `rosa` CLI for your operating system.
+.. Extract the `rosa` binary file from the downloaded archive. The following example extracts the binary from a Linux tar archive:
++
+[source,terminal]
+----
+$ tar xvf rosa-linux.tar.gz
+----
+.. Add `rosa` to your path. In the following example, the `/usr/local/bin` directory is included in the path of the user:
++
+[source,terminal]
+----
+$ sudo mv rosa /usr/local/bin/rosa
+----
+.. Verify if the `rosa` CLI tool is installed correctly by querying the `rosa` version:
++
+[source,terminal]
+----
+$ rosa version
+----
++
+.Example output
+[source,terminal]
+----
+1.1.7
+----
++
+.. Optional: Generate the command completion scripts for the `rosa` CLI. The following example generates the Bash completion scripts for a Linux machine:
++
+[source,terminal]
+----
+$ rosa completion bash | sudo tee /etc/bash_completion.d/rosa
+----
+.. Optional: Source the scripts to enable `rosa` command completion from your existing terminal. The following example sources the Bash completion scripts for `rosa` on a Linux machine:
++
+[source,terminal]
+----
+$ source /etc/bash_completion.d/rosa
+----
+.. Log in to your Red Hat account by using the `rosa` CLI:
++
+[source,terminal]
+----
+$ rosa login
+----
++
+.Example output
+[source,terminal]
+----
+To login to your Red Hat account, get an offline access token at https://console.redhat.com/openshift/token/rosa
+? Copy the token and paste it here: 
+----
++
+Navigate to the URL listed in the command output to obtain an offline access token. Specify the token at the CLI prompt to log in.
++
+[NOTE]
+====
+You can subsequently specify the offline access token by using the `--token="<offline_access_token>"` argument when you run the `rosa login` command.
+====
+.. Verify if you are logged in successfully and check your credentials:
```

Comment:
"Verify that...?"

Reply:
I will update this line after we have reviewed this topic in the first of these threads.

Reply:
Based on our subsequent conversation in Slack, I will leave the line as "verify if".